console/src/components/UIKit/atoms/SwitchButton/SwitchButton.js

```diff
@@ -0,0 +1,73 @@
+import styled, { css } from 'styled-components';
+
+export const StyledSwitchButton = styled.div`
+  label {
+    position: relative;
+    display: inline-block;
+    width: 40px;
+    height: 20px;
+
+    input {
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    input:checked {
+      background-color: #1fd6e5;
+    }
+
+    input:focus {
+      box-shadow: 0 0 1px #1fd6e5;
+    }
+
+    input:checked {
+      -webkit-transform: translateX(20px);
+      -ms-transform: translateX(20px);
+      transform: translateX(20px);
+    }
+  }
+`;
+
+export const StyledSlider = styled.span`
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #484538;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+  border-radius: 34px;
+
+  &:before {
+    border-radius: 50%;
+    position: absolute;
+    content: '';
+    height: 16px;
+    width: 16px;
+    left: 2px;
+    bottom: 2px;
+    background-color: white;
+    -webkit-transition: 0.4s;
+    transition: 0.4s;
+  }
+
+  &:hover {
+    box-shadow: 0 0 1px #1fd6e5;
+  }
+
+  ${({ checked }) =>
+    checked &&
```

Comment:
Can you refactor this to ternary so we won't have `false` here?
```js
${({ checked }) =>
    checked ? css`` : ''
```

Reply:
ok